packages/pluggableWidgets/dropdown-sort-web/src/components/__test__/SortComponent.spec.tsx

```diff
@@ -71,4 +76,97 @@ describe("Sort selector", () => {
             expect(component.find("input").first().prop("value")).toBe(defaultOptions[2].caption);
         });
     });
+
+    describe("focus", () => {
+        beforeEach(() => (document.body.innerHTML = ""));
+
+        it("changes focused element when pressing the input", () => {
+            const component = renderTestingLib(
+                <SortComponent options={defaultOptions} emptyOptionCaption="Click me" />
+            );
+
+            expect(document.body).toHaveFocus();
+            const input = component.getByPlaceholderText("Click me");
+            expect(input).toBeDefined();
+            fireEvent.click(input);
+
+            jest.advanceTimersByTime(10);
+
+            const items = component.getAllByRole("menuitem");
+            expect(items[0]).toHaveFocus();
+        });
+
+        it("changes focused element back to the input when pressing shift+tab in the first element", () => {
+            const component = renderTestingLib(
+                <SortComponent options={defaultOptions} emptyOptionCaption="Click me" />
+            );
+
+            expect(document.body).toHaveFocus();
+
+            const input = component.getByPlaceholderText("Click me");
+            expect(input).toBeDefined();
+            fireEvent.click(input);
+
+            jest.advanceTimersByTime(10);
+
+            const items = component.getAllByRole("menuitem");
+            expect(items[0]).toHaveFocus();
+
+            userEvent.tab({ shift: true });
+
+            jest.advanceTimersByTime(10);
+
+            expect(input).toHaveFocus();
+        });
+
+        it("changes focused element back to the input when pressing tab on the last item", () => {
+            const component = renderTestingLib(
+                <SortComponent options={[{ caption: "1", value: "_1" }]} emptyOptionCaption="Click me" />
+            );
+
+            expect(document.body).toHaveFocus();
+
+            const input = component.getByPlaceholderText("Click me");
+            fireEvent.click(input);
+
+            jest.advanceTimersByTime(10);
+
+            const items = component.getAllByRole("menuitem");
+            expect(items[0]).toHaveFocus();
+
+            userEvent.tab();
+            expect(items[1]).toHaveFocus();
+            userEvent.tab();
+
+            jest.advanceTimersByTime(10);
+
+            expect(input).toHaveFocus();
+        });
+
+        it("changes focused element back to the input when pressing escape on the last item", () => {
```

Comment:
Only on the last item? Wouldn't it make sense for this to be on all? 

If not, can we get a test on the other. If so, can you change the name of the test?

Reply:
Nope, it works in any items.. was just because in this case was the last.

Reply:
Fixed